packages/i18n/src/store/index.ts

```
@@ -130,51 +130,56 @@ export class TranslationStore {
+   * Helper function to import and merge multiple translation files for a language
+   * @param language - The language code
+   * @param files - Array of file names to import (without .json extension)
+   * @returns Promise that resolves to merged translations
+   */
+  private async importAndMergeFiles(language: TLanguage, files: string[]): Promise<any> {
+    const importPromises = files.map((file) => import(`../locales/${language}/${file}.json`));
+
+    const modules = await Promise.all(importPromises);
+    const merged = modules.reduce(
+      (acc, module) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)